li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerConfig.java

```diff
@@ -92,6 +93,10 @@
 
   private static final String MAX_POLL_RECORDS_DOC = "The maximum number of records returned in a single call to poll().";
 
+  private static final String DEFAULT_API_TIMEOUT_MS_DOC = "Specifies the timeout (in milliseconds) for consumer APIs" +
```

Comment:
we inherit this from upstream, right ?

Reply:
k, i was slightly confused, but can we call this max.block.ms? since thats an equivalent config that already exists for upstream producers.

Reply:
This is what upstream consumer is already using as the default timeout value.